docs/dev/e2eTesting-GettingStarted.md

```diff
@@ -0,0 +1,96 @@
+# E2E Testing
+## Getting Started
+
+This document describes the steps that need to be taken to run a e2e test from your machine that points to a 
```

Comment:
Talking about the `pre-configured` network, is there any doc to describe how to create our own pre-configured network, or we normally always use existing network?

Reply:
You can, But in an End to End testing perspective.. you should also configure a mirror node for your custom network.

Reply:
Is there anything special to create/configure a mirror node?

Reply:
Mirror node can be ran easy enough, but to create a end to end network is not easy due to the need for ops uploaders and s3 buckets.

Reply:
I see. Thanks for the explanation.